Package.swift

```diff
@@ -18,6 +18,29 @@
 
 import PackageDescription
 
+#if os(Linux)
```

Comment:
this change will break cross-compilation. Generally in `Package.swift` files there should be no `#if os(...)`/`#if arch(...)` etc. I think the right change would be to make `CLinuxHelpers` compile on all OSes by using `#if __linux` etc in the .c/.h files similar to what SwiftNIO does for its `CNIOLinux` module: https://github.com/apple/swift-nio/tree/master/Sources/CNIOLinux

Reply:
Maybe you are right)
I rollback changes